.env

```diff
@@ -1,4 +1,4 @@
-API_KEY="api_key"
```

Comment:
thank you for adding the .gitignore, but just change the api key string back to "api_key"